app/components/NoticeFormat.tsx

```diff
@@ -61,6 +66,20 @@ export function NoticeFormatInput({
         </>
       ),
     },
+    ...(location.pathname !== '/user/email/edit'
```

Comment:
Add a FIXME comment to explain why we are checking the location here.

Better yet… just add a showJson prop to the component to explicitly control whether it is visible.